Introduction/Introduction.md

```diff
@@ -0,0 +1,23 @@
+Kernel introduction.
+================================================================================
+
+The Begining
+--------------------------------------------------------------------------------
+Linux has began in the Operating system more important in the last years and many people use it in their lives commonly.
+But nobody like remember the history about the Linux kernel.
+
+It s now installed on millions of systems and used by home users and professionals alike for a wide range of tasks. From miniature embedded systems in wristwatches to massively parallel mainframes, the are countless way of exploiting linux productively.
+
+This the first part to undestand if you want to begin with this topics.
+Linux looks like any other Unix systems; indeed, UNIX compatiblity has a major design and goal of the Linux Kernel project and it is considered as the core of the Operating Systems.
+The First difference betwen any other programs in the system consist that controls the hardware and the applications but the applications do not communicate with the hardware directly, instead the go to the kernel. In summary
+
+In 1991 a Finish student named of Linus Benedict Torvalds begin with his development. He released the first linux version on September 1991, and on february 1992, he lisensed the kernel under the GPL licence.
```

Comment:
- Fi**n**nish
- named <s>of</s> Linus (not **of**)
- beg**a**
- **L**inux (capitalized)
- **In** September. (Also, it was September 25 if memory serves me correctly.)
- Period after 1991 -- start a new sentence.
- li**c**ensed